packages/moonstone/styles/internal/fonts.less

```diff
@@ -33,20 +34,20 @@
 // In the case of glyphs existing in two fonts, the later glyphs will be used.
 @fonts: {
 	am: local("LG Smart UI Amharic");
-	as: local("LG Smart UI Bengali"); // same as `bn`
-	bn: local("LG Smart UI Bengali");
-	en-JP: local("LG Smart UI JP"), local("LG Display_JP_Bold") 700; // same as `ja`
```

Comment:
Please do not remove `en-JP`. It should match `ja` below.

Reply:
I think it is repeated code because `en-JP` is handled in `fontGenerator.js` and Engilsh and Japanese should be printed with `LG Smart UI` except in Japan locales(`en-JP`, `ja-JP`). Could you please explain to me why we should keep this line?

Reply:
This is not repeated code. This establishes the language font into the base "Moonstone" font and adds a "Moonstone en-JP" font family in case an app needs to refer to a specific locale's font family. `fontGenerator` only creates fonts at runtime linked to a locale-switching event.

Historically, all Japanese text should use this font. That's why this code exists. The addition of a more complete `LG Smart UI` font means that these requirements have changed. This can be removed if and only if the following are both true:
* We do not want apps to be able to reference the Japanese font directly (via `font-family: "Moonstone en-JP";`)
* We do not want these glyphs being inserted into the main/common font: "Moonstone".

If we do want the first bullet above, this code must be retained. The `LG Smart UI` font being applied on top of these will override any previous existing glyph with its own, so if that font does indeed have all of the same Japanese and english codepoints covered, this will be fully overwritten and everything will look correct. Also, the "Moonstone en-JP" font will still exist for any app currently referencing it.